cmd/evm/runner.go

```diff
@@ -91,7 +91,7 @@ func timedExec(bench bool, execFunc func() ([]byte, uint64, error)) (output []by
 		// There are other useful stats here that could be reported.
 		stats.time = time.Duration(result.NsPerOp())
 		stats.allocs = result.AllocsPerOp()
-		stats.bytesAllocated = result.AllocedBytesPerOp()
+		stats.bytesAllocated = result.AllocatedBytesPerOp()
```

Comment:
The function definition is here 
![image](https://github.com/taikoxyz/taiko-geth/assets/160697994/0f161f73-cffc-4d57-ac59-10552b7a2523)